tests/unit/lms/services/lti_registration_test.py

```diff
@@ -10,6 +10,9 @@ class TestLTIRegistrationService:
     def test_get(self, svc, registration):
         assert svc.get(registration.issuer, registration.client_id) == registration
```

Comment:
If we can look it up by `issuer` alone in the test below, do we actually prove that the client id is used at all? Does this test require a noise element with the same issuer and a different client?